docs/src/apimanual.md

```diff
@@ -931,20 +932,60 @@ between `y_i` and the vector of scalar-valued quadratic functions.
 
 ### Automatic reformulation
 
+#### Variable reformulation
+
+A variable is often created in a set unsupported by the solver while it could be
```

Comment:
Use the existing terminology around constrained variables. It's not obvious what "creating a variable in a set" means.